frame/bounties/src/lib.rs

```diff
@@ -548,9 +549,13 @@ decl_module! {
 					let balance = T::Currency::free_balance(&bounty_account);
 					let fee = bounty.fee.min(balance); // just to be safe
 					let payout = balance.saturating_sub(fee);
-					let _ = T::Currency::unreserve(&curator, bounty.curator_deposit);
-					let _ = T::Currency::transfer(&bounty_account, &curator, fee, AllowDeath); // should not fail
-					let _ = T::Currency::transfer(&bounty_account, &beneficiary, payout, AllowDeath); // should not fail
+					let err_amount = T::Currency::unreserve(&curator, bounty.curator_deposit);
+					debug_assert!(err_amount.is_zero());
+					let res = T::Currency::transfer(&bounty_account, &curator, fee, AllowDeath); // should not fail
```

Comment:
Also these Lines exceed 100, but they did before bc. of the comment, so I didn't change it.